dat/maps/mt_elbrus/mt_elbrus_path3_script.lua

```diff
@@ -597,17 +597,17 @@ function _CreateEvents()
     EventManager:RegisterEvent(orlinn_move_next_to_hero_event1);
 
     dialogue = vt_map.SpriteDialogue();
-    text = vt_system.Translate("This place is the village's old cemetery, used when the former villagers lived in Layna...");
+    text = vt_system.Translate("This place is the village's old cemetery. It was used by the former villagers who lived in Layna.");
     dialogue:AddLine(text, kalya);
     text = vt_system.Translate("The former villagers? What do you mean?");
     dialogue:AddLineEmote(text, hero, "exclamation");
-    text = vt_system.Translate("Well, Layna Village was abandoned a long time ago, before your parents and all the others settled here... Your mother never told you that?");
+    text = vt_system.Translate("Layna Village was abandoned a long time ago, before your parents and all the others settled here. Your mother never told you that?");
     dialogue:AddLineEventEmote(text, kalya, "Kalya looks at Bronann", "", "thinking dots");
-    text = vt_system.Translate("I've never been further alone before... This place gives me the chill...");
+    text = vt_system.Translate("I've never been more alone before. This place gives me the chills.");
```

Comment:
I'm revealing my lack of familiarity with the story =p

It's very unusual in America for children to refer to their parents by their first name. It's more common for children to refer to adoptive or step-parents by their first name. Would Kalya say "Herth" instead of "my father"?